.env.template

```diff
@@ -1 +1,2 @@
-MYSQL_ROOT_PASSWORD=password
\ No newline at end of file
+MYSQL_ROOT_PASSWORD=password
+MEMCACHED_CACHE_SIZE=64
```

Comment:
Unità di misura? Magari specifica in un commento (eventualmente link a docu)

Reply:
ho aggiunto l'unità di misura, se serve aggiungo il link nel readme